test/java_rules/java_toolchain/src/main/java/net/thoughtmachine/BUILD

```diff
@@ -1,4 +1,4 @@
-if CONFIG.OS == "linux" or CONFIG.OS == "darwin":
+if is_platform(os = "linux") or is_platform(os = "darwin"):
```

Comment:
```
if is_platform(os = ["linux", "darwin"]):
```
WDYT?

Reply:
Sounds like it'd be more useful that way :D